### DISCUSSION TITLE
UBL Mesh Tilt without probe?

### DISCUSSION THREAD

**Discussion by melvinmicallef** [Jyers/Marlin#337] [Q&A] [answered]
Is it possible to perform the mesh tilt feature to tilt the mesh by setting 4 points manually (without a probe)? This would avoid redoing the whole mesh. Thank you

**Comment by Jyers** [comment, marked as answer]
I don't know if marlin supports manual mesh tilting, as from what I can tell it is designed as an automatic operation. I will definitely look into it though to see if this may be possible!
